main/domoticz.cpp

```diff
@@ -124,12 +124,12 @@ std::string szWebRoot;
 std::string dbasefile;
 
 bool bHasInternalTemperature=false;
-std::string szInternalTemperatureCommand = "/opt/vc/bin/vcgencmd measure_temp";
+std::string szInternalTemperatureCommand = "vcgencmd measure_temp";
```

Comment:
@akamming , maybe separate the command from the arguments? Have a define at the top for the command and use this define here.